### PR TITLE
Rebuild lld 14.0.6 with libxml2 2.10

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Actions:
1. Bump build number
2. Reorder dependencies
3. Add `host` pinnings
4. Add a `description`

### CODE REVIEW

**Review by cbouss — APPROVED** [review]

**Comment by skupr-anaconda** [comment]
llvmdev https://github.com/AnacondaRecipes/llvmdev-feedstock/pull/10 has missing ppc64le artifacts after merging. Waiting for retriggering the CI flow

**Review by varlackc — COMMENTED** [review]
Just curious to know if we can to remove the `abs.yaml` file before merging, in case that we can not do it due to issues with the abs system, I am curious if it might be a good idea to remove the channel information from the `abs.yaml`, since technically the package will be sent to defaults.

**Comment by skupr-anaconda** [comment]
> Just curious to know if we can to remove the `abs.yaml` file before merging, in case that we can not do it due to issues with the abs system, I am curious if it might be a good idea to remove the channel information from the `abs.yaml`, since technically the package will be sent to defaults.

It was removed

**Pull request merged**
